source/CleoBase.cpp

```diff
@@ -252,9 +252,9 @@ namespace CLEO
         GetInstance().RemoveCallback(id, func);
     }
 
-    DWORD WINAPI CLEO_GetInternalAudioStream(CLEO::CRunningScript* thread, DWORD stream) // arg CAudioStream *
+    DWORD WINAPI CLEO_GetInternalAudioStream(CLEO::CRunningScript* unused, DWORD audioStreamPtr)
     {
-        return stream; // CAudioStream::streamInternal offset is 0
+        return audioStreamPtr + 0x8; // offsetof(CAudioStream, streamInternal)
```

Comment:
why can't you do `audioStreamPtr->GetInternal()` here, like CLEO4?

Reply:
Audio stuff is now separated plugin. I do not think it is worth to create project dependency in that direction to support just this single export.

Reply:
Offset of internal BASS stream in relation to audio stream handle returned by CLEO is untouchable, as some scripts are using it already, just by adding offset and reading memory.
The export was apparently added by Junior for purposes of CLEO+ opcodes.

Reply:
so why do you need this class Dummy then? what offset the scripts use?